src/fdcache_fdinfo.h

```diff
@@ -81,10 +81,11 @@ class PseudoFdInfo
         bool ResetUploadInfo() REQUIRES(upload_list_lock);
         bool RowInitialUploadInfo(const std::string& id, bool is_cancel_mp);
         bool CompleteInstruction(int result) REQUIRES(upload_list_lock);
-        bool ParallelMultipartUpload(const char* path, const mp_part_list_t& mplist, bool is_copy) REQUIRES(upload_list_lock);
+        bool ParallelMultipartUpload(const char* path, const mp_part_list_t& mplist, bool is_copy);
         bool InsertUploadPart(off_t start, off_t size, int part_num, bool is_copy, etagpair** ppetag) REQUIRES(upload_list_lock);
         bool CancelAllThreads();
         bool ExtractUploadPartsFromUntreatedArea(const off_t& untreated_start, const off_t& untreated_size, mp_part_list_t& to_upload_list, filepart_list_t& cancel_upload_list, off_t max_mp_size);
+        bool IsUploadingWitoutLock() const REQUIRES(upload_list_lock);
```

Comment:
This is misspelled but let's use a consistent naming convention with the other code.  The methods that lock should be named `Foo` and the methods that require the caller to have the lock should be named `FooHasLock`.  I am happy to debate the best naming for this in a separate PR but this PR should follow the existing practice.

Reply:
We will follow the HasLock naming convention.(Also, this is a mispelling.)

Reply:
Fixed.